test/src/UnitTests.cpp

```diff
@@ -0,0 +1,41 @@
+// Note: No need to increase compile time by including UnitTests.hpp
+#include <SFML/System/String.hpp>
+#include <SFML/System/Time.hpp>
+#include <SFML/Window/VideoMode.hpp>
+#include <SFML/Graphics/Color.hpp>
+#include <sstream>
+
+// String conversions for Catch framework
+namespace Catch
+{
+    std::string toString(const sf::String& string)
+    {
+        return string.toAnsiString();
+    }
+
+    std::string toString(sf::Time time)
+    {
+        std::ostringstream stream;
+        stream << time.asMicroseconds() << "us";
```

Comment:
Won't we need milliseconds more frequently? Microseconds are too noisy for human readable output.

Reply:
The point isn't necessarily human readable output, but predictable and comparable output. Microseconds is the smallest unit and what's saved internally, so this makes the most sense.